content/docs/2.4/scalers/influxdb.md

```diff
@@ -30,18 +30,18 @@ triggers:
 
 **Parameter list:**
 
-- `authToken` field holds the authentication token needed for the InfluxDB client to communicate with an associated server. 
+- `authToken` field holds the authentication token needed for the InfluxDB client to communicate with an associated server. Can be sourced from `TriggerAuthentication` as well.
 - `authTokenFromEnv` defines the authorization token, similar to `authToken`, but reads it from an environment variable on the scale target.
-- `organizationName` is the organization name needed for the client to locate all information contained in that [organization](https://docs.influxdata.com/influxdb/v2.0/organizations/) such as buckets, tasks, etc.
+- `organizationName` is the organization name needed for the client to locate all information contained in that [organization](https://docs.influxdata.com/influxdb/v2.0/organizations/) such as buckets, tasks, etc. Can be sourced from `TriggerAuthentication` as well.
 - `organizationNameFromEnv` defines the organization name, similar to `organizationName`, but reads it from an environment variable on the scale target.
-- `serverURL` holds the url value of the InfluxDB server.
+- `serverURL` holds the url value of the InfluxDB server. Can be sourced from `TriggerAuthentication` as well.
 - `thresholdValue` is provided by the user. This value can vary from use case to use case depending on the data of interest, and is needed to trigger the scaling in/out depending on what value comes back from the query.
 - `query` is the flux query that will yield the value for the scaler to compare the `thresholdValue` against.
 - `metricName` an optional name to assign to the metric. If not set KEDA will generate a name based on masked version of the server url and organization name. If using more than one trigger it is required that all `metricName`(s) be unique.
 
 ### Authentication Parameters
 
-You can authenticate by using an authorization token.
+You can authenticate by using an authorization token. You can also specify `serverURL`, `organizationName`, and `authToken` via a `TriggerAuthentication` or `ClusterTriggerAuthentication` and then specify this via `authenticationRef`. See [Authentication](https://keda.sh/docs/2.4/concepts/authentication/#re-use-credentials-and-delegate-auth-with-triggerauthentication) for more details.
```

Comment:
Can you extend the list below please?

Reply:
With that, I mean just adding them here: https://github.com/kedacore/keda-docs/pull/472/files#diff-2676ca2c8d04843d6214bff72f1364e2f8991c188e522e15ffe8ee3a81314c85R48